.github/workflows/basic.yml

```diff
@@ -204,21 +204,23 @@ jobs:
             os: macos-latest
             preset: defaults
             install: true
+            # Undefined-behavior in leak detection breaks #include<list> on Mac
+            cmake_args: -DCPPUTEST_MEM_LEAK_DETECTION_DISABLED=YES
```

Comment:
I think disabling the memory leak detection here does not seem the right thing to do.

Reply:
This change was from #1805. I needed it to get a passing pipeline. I've removed it from this MR.